cwd/echo/echo.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	s "strings"
+)
+
+func main() {
+	var showLines bool
+	flag.BoolVar(&showLines, "n", false, "Omit new line")
+	flag.Parse()
+	args := s.Join(flag.Args(), " ")
+	if showLines {
+		fmt.Print(args)
```

Comment:
this does not implement the requirement of echo -n flag

![image](https://github.com/codescalersinternships/Coreutils-Mariam_Mahrous/assets/67752395/eb0369a6-9a18-46e8-ad2f-4477671c21c0)